packages/web-component-library-react/package.json

```diff
@@ -29,7 +29,7 @@
     "dist/"
   ],
   "devDependencies": {
-    "@types/node": "16.11.19",
+    "@types/node": "17.0.8",
```

Comment:
Let's include this package in the `npm-check-updates`/`dependabot.yml`, since Node.js is upgraded manually and we aim to keep the Node.js types in sync with the actual Node.js version. Incorrect version here could lead to errors about us using APIs that have changed in an upcoming version.